src/sentry/static/sentry/app/components/group/externalIssuesList.jsx

```diff
@@ -72,26 +72,34 @@ class ExternalIssueList extends AsyncComponent {
   fetchSentryAppData() {
     const {api, orgId, group, project} = this.props;
 
-    api
-      .requestPromise(
-        `/organizations/${orgId}/sentry-app-components/?filter=issue-link&projectId=${project.id}`
-      )
-      .then(data => {
-        this.setState({components: data});
-      })
-      .catch(error => {
-        return;
-      });
-
-    api
-      .requestPromise(`/groups/${group.id}/external-issues/`)
-      .then(data => {
-        ExternalIssueStore.load(data);
-        this.setState({externalIssues: data});
-      })
-      .catch(error => {
+    if (project && project.organization) {
```

Comment:
I don't think `organization` should exist at all on project. I think you'll want to use `withOrganization` to get org from props?

Reply:
Gotcha. It does locally for me. Not sure where it's coming from, though.

Reply:
(btw we figured it out and it's because of sentry10)